### PR TITLE
Create Venus

### PR DESCRIPTION
**Pull request by chichiwepe**
Trying a pull request

### CODE REVIEW

File: ABC-challenge/photos/Venus [hunk 1]
```diff
@@ -0,0 +1 @@
+V is for venus
```
Comment (LazaroAlsina) [inline ABC-challenge/photos/Venus:1]: This looks good

**Review by z-shug — APPROVED** [review]

**Comment by z-shug** [comment]
Hey Laz can we get an approval

**Comment by LazaroAlsina** [comment]
@z-shug What do you mean?

**Comment by z-shug** [comment]
@LazaroAlsina We need two people to approve

**Comment by LazaroAlsina** [comment]
Got it, let me approve this

**Review by LazaroAlsina — APPROVED** [review]
I approve

**Pull request merged**